src/lib/i18n.json

```diff
@@ -20,6 +20,9 @@
     "setup-cmd-prompt-err-no-subscriptions": "No subscriptions found.",
     "setup-cmd-prompt-err-subscription-missing": "Subscription Identifier was missing.",
     "setup-cmd-prompt-err-input-file-missing": "{0} did not exist or not accessible. Make sure that it is accessible.",
+    "setup-cmd-core-api-err": "Could not get Core API client. Check the Azure credential",
```

Comment:
Core API is also for Azure DevOps. Let's be specific so we don't confuse the user between Azure and Azure DevOps here.

Reply:
can you suggest a good message? thanks

Reply:
The same message as the other errors:

Something like 
`Could not initialize Core API client. Check the Azure DevOps credentials.`

This is just being explicit about Azure vs Azure DevOps

Reply:
actually, this one is for Azure and not Azure DevOps. the other 2 are for Azure DevOps. We are using PAT to authenticate for this one and using Service Principal for the other 2.